src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -977,21 +1041,182 @@ public void TestRuntimeEnvironmentVariable()
             ValidateCosmosDbSetup(server);
         }
 
-        [TestMethod("Validates the runtime configuration file."), TestCategory(TestCategory.MSSQL)]
-        public void TestConfigIsValid()
+        /// <summary>
+        /// This method tests the config properties like data-source, runtime settings and entities.
+        /// </summary>
+        [TestMethod("Validates the runtime configuration file properties."), TestCategory(TestCategory.MSSQL)]
+        public void TestConfigPropertiesAreValid()
         {
             TestHelper.SetupDatabaseEnvironment(MSSQL_ENVIRONMENT);
             FileSystemRuntimeConfigLoader configPath = TestHelper.GetRuntimeConfigLoader();
             RuntimeConfigProvider configProvider = TestHelper.GetRuntimeConfigProvider(configPath);
 
             Mock<ILogger<RuntimeConfigValidator>> configValidatorLogger = new();
-            IConfigValidator configValidator =
-                new RuntimeConfigValidator(
+            RuntimeConfigValidator configValidator =
+                new (
                     configProvider,
                     new MockFileSystem(),
                     configValidatorLogger.Object);
```

Comment:
you could provide _isValidateOnly property as you construct this object RuntimeConfigValidator here.